testsuite/features/secondary/minssh_action_chain.feature

```diff
@@ -187,7 +187,7 @@ Feature: Salt SSH action chain
   Scenario: Delete the action chain for SSH minion
     When I follow the left menu "Schedule > Action Chains"
     And I follow "new action chain"
-    And I follow "delete action chain" in the content area
+    And I follow "delete action" in the content area
```

Comment:
I'm not sure this is correct, we name "action chain" in the menu, and we name "action chain" when we are creating it, why we name it only "action" when we are deleting it? That's not consistent.

Also, the title of this scenario is not consistent. Deleting an action should be different than deleting a chain of actions.